packages/authentication/modules/access/useLogin/index.ts

```
@@ -21,14 +22,28 @@
 import {
   isLoginChangeExpiredPasswordRedirectResponse,
   isLoginMfaResponse,
 } from '../../../utils/login'
 import { CODE_VALIDATION_INITIAL_VALUES, CODE_VALIDATION_SCHEMA } from '../../mfa/constants'
+import { MinimalProfile, useCurrentProfile } from '../../profile'
 import { DEFAULT_INITIAL_VALUES, DEFAULT_VALIDATION_SCHEMA } from './constants'
 import type { UseLoginOptions } from './types'
 
+const makeImagePathAbsolute = (profileArg: MinimalProfile | null) => {
+  const profile = profileArg
+  if (profile && profile.image) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)